frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -15,10 +15,10 @@ interface TableProps {
 
 interface LineChartTableRowData {
   period: number;
-  count: number;
-  value: number;
-  lower: number;
-  upper: number;
+  count: number | null;
+  value: number | null;
+  lower: number | null;
+  upper: number | null;
```

Comment:
Could this not use optional properties?
```suggestion
  count?: number;
  value?: number;
  lower?: number;
  upper?: number;
```

Reply:
Fixed as suggested